### DISCUSSION TITLE
IP proxy options

### DISCUSSION THREAD

**Discussion by luisvivasg** [deta/cloud-docs#653] [Help] [answered]
I trying to scrape a website. After 2 sucessful requests, I got a captcha. When I tried with locally I worked fine with the sleeps that I added, but now it seems to recognize the ip as a server.  Are these Amazon ip? I there a way to easily rotate the ip of the micro?

Thanks

**Comment by AyushSehrawat** [comment, marked as answer]
Yes these are ips from aws pool of ips, and the ip of micro rotates very often (iirc every few minutes), but there is no option to manually change ip :).
Try something else to skip the captcha ( might work well locally since it detected its normal unlike micros who have ips from aws pool)
